src/main/java/uk/gov/hmcts/divorce/common/notification/ApplicationIssuedNotification.java

```diff
@@ -141,13 +146,26 @@ public void sendToApplicant2(final CaseData caseData, final Long caseId) {
     @Override
     public void sendToApplicant2Solicitor(final CaseData caseData, final Long caseId) {
 
-        log.info("Sending Notice Of Proceedings email to respondent solicitor.  Case ID: {}", caseId);
+        if (caseData.getApplicationType().isSole()) {
+            if (isNull(caseData.getApplicant2().getSolicitor().getOrganisationPolicy())) {
+                log.info("Sending copy of D10 and coversheet to respondent solicitor for case: {}", caseId);
+                d10Printer.printD10WithCoversheet(caseData, caseId);
+            }
+
+            log.info("Sending Notice of Proceedings letter to respondent solicitor for case: {}", caseId);
+            noticeOfProceedingsPrinter.sendLetterToApplicant2Solicitor(caseData, caseId);
+
+            log.info("Sending copy of Divorce Application to respondent solicitor for case: {}", caseId);
+            applicationPrinter.sendDivorceApplicationPdf(caseData, caseId);
```

Comment:
We normally don't send 3 separate requests to bulk print. This will end up in producing 3 different letters being printed/posted to the solicitor